cleanlab/regression/rank.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np 
+
+def get_label_quality_score(
+    true_labels: np.ndarray,
```

Comment:
CAUTION: never use "true_labels" in any cleanlab code. We do not have true labels in any setting where cleanlab is typically applied. These should simply be called: `labels`
(you can also refer to them as "given labels" in documentation and tutorial text)